cpp/ql/src/semmlecode.cpp.dbscheme

```diff
@@ -1190,6 +1190,25 @@ funbind(
                     | @delete_expr
                     | @delete_array_expr;
 
+
+@prefix_crement_oper_expr = @preincrexpr | @predecrexpr
+
+@postfix_crement_oper_expr = @postincrexpr | @postdecrexpr
+
+@incr_oper_expr = @preincrexpr | @postincrexpr
+
+@decr_oper_expr = @predecrexpr | @postdecrexpr
+
+@crement_op_expr = @incr_oper_expr | @decr_oper_expr | @prefix_crement_oper_expr | @postfix_crement_oper_expr
```

Comment:
```suggestion
@crement_op_expr = @incr_oper_expr | @decr_oper_expr
```

These two together cover all four "crement" operations, right?

Reply:
This is correct. I'll accept this commit suggestion!